### PR TITLE
swapon parser (#383)

### PR DESCRIPTION
**Pull request by r-darwish**

### CODE REVIEW

**Review by r-darwish — COMMENTED** [review]
Still missing unit tests and I wasn't sure which scripts update the documentation. Let me know if it looks good enough to proceed with what's left to do.

File: jc/lib.py [hunk 1]
```diff
@@ -9,243 +9,248 @@
 from jc import appdirs
 
 
-__version__ = '1.23.6'
```
Comment (r-darwish) [inline jc/lib.py:12]: Should have saved without formatting. I'll fix this

File: jc/parsers/swapon.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    Returns:
+
+        Dictionary. Raw or processed structured data.
+    """
+    jc.utils.compatibility(__name__, info.compatible, quiet)
+    jc.utils.input_type_check(data)
+
+    raw_output = list[dict]()
+
+    if jc.utils.has_data(data):
```
Comment (r-darwish) [inline jc/parsers/swapon.py:154]: Need for fix `swapon --output-all`

**Comment by kellyjonbrazil** [comment]
Looking great - thanks for the contribution!

**Comment by r-darwish** [comment]
No problem. I just pushed a fix and made sure it works in Python 3.6

**Comment by kellyjonbrazil** [comment]
Looks great! Thanks again for the contribution. I'll get this in the next release.

**Pull request merged**